.github/CONTRIBUTING.md

```diff
@@ -67,4 +67,31 @@ The body should include the motivation for the change and contrast this with pre
 The footer should contain any information about **Breaking Changes** and is also the place to
 reference GitHub issues that this commit **Closes**.
 
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
\ No newline at end of file
+**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+
+## Changelog Guidelines
+All changes will be documented using a [CHANGELOG](https://github.com/rakutentech/android-miniapp/blob/master/CHANGELOG.md) by writing chronologically for each version with the following format:
+
+```
+Release Version (YYYY-MM-DD)
+SDK
+- <description>
+<BLANK LINE>
+Sample App
+- <description>
+```
+
+### Description
+A sentence describes the type of change with an information.
+
+```
+<bold><change type></bold>: <subject>
+```
+
+#### Change type
+* **Feature**: Added a new feature.
+* **Fixed**: When there is a hotfix or bugfix. Something has been upgraded could be mentioned too.
+* **Deprecated**: Still available for use but will no longer be developed or supported. This is an indication it should not be used and will be removed in the next major version release.
+* **Obsoleted**: No longer works as expected since a better alternative is available and will maybe discontinued from the next releases.
```

Comment:
What is the use case of `Obsoleted`? I think maybe this can be removed.

Reply:
`Obsoleted` here is a previous step of `Removed` phase which means kinda out-of-use. I don't find any "Obsoleted" annotation in Android by default yet, rather Android uses [Deprecated](https://developer.android.com/reference/java/lang/Deprecated) to define the situation. 

in iOS, I am not sure if we need to use `Obsoleted`. 
cc: @Climbatize @ts-leojoseph-r